src/track/trace_unwind_tables.cpp

```diff
@@ -0,0 +1,40 @@
+/**
```

Comment:
Should I put any copyright notice here? If so, what it should be exactly?

Reply:
the same as for trace.h but with your name instead of mine and 2019 as year